### PR TITLE
Fetch project owner on get

### PR DESCRIPTION
**Pull request by jorwoods**
Adds `owner_id` to `ProjectItem` and populates it via the `projects.get` endpoint. Adds stubs for updating owner via create/update calls, but leaves them commented out pending server-side enhancement. Raises a `NotImplementedError` when trying to set `owner_id` on a `ProjectItem` as that is currently not supported server side.

### CODE REVIEW

**Comment by t8y8** [comment]
I think grabbing owner and populating it is great, the NotImplemented exception is fine but I'm not a fan of the comments hanging out in the code.

Would you be willing to nix the comments but keep the rest :)

**Comment by jorwoods** [comment]
I can understand not liking the commented code there. I did it with the idea of referencing what needs to be enabled once the issue is resolved, but removed it.

File: tableauserverclient/models/project_item.py [hunk 1]
```diff
@@ -18,6 +18,7 @@ def __init__(self, name, description=None, content_permissions=None, parent_id=N
         self.name = name
         self.content_permissions = content_permissions
         self.parent_id = parent_id
+        # self.owner_id = owner_id
```
Comment (t8y8) [inline tableauserverclient/models/project_item.py:21]: Missed one :)

**Review by t8y8 — CHANGES_REQUESTED** [review]

File: tableauserverclient/models/project_item.py (hunk 1 above)
Comment (jorwoods) [reply tableauserverclient/models/project_item.py]: Pushed patch

**Review by t8y8 — APPROVED** [review]
🚀

**Comment by shinchris** [comment]
Thanks @jorwoods!

**Pull request merged**